src/utils/text.js

```diff
@@ -0,0 +1,147 @@
+
+export const viewActionText = 'View'
+
+export const updateActionText = 'Update'
+
+export const newLessonSubmissionDescriptionText = 'Have an idea for an egghead lesson? Fill out this information and get started. Feel free to submit as many ideas as you like.'
+
+export const loadingTitleText = 'Loading'
+
+export const lessonTitleLabelText = 'Title *'
+export const lessonTechnologyLabelText = 'Technology *'
+export const lessonSummaryLabelText = 'Summary'
+
+export const previousLabelText = 'Previous'
+export const nextLabelText = 'Next'
+
+export const claimDescriptionText = 'Claim a new lesson to get started'
+export const claimCompleteDescriptionText = 'This lesson is now yours. The next step is to record.'
+
+export const instructorPulseTitleText = 'Instructor Pulse'
+
+export const newRequestedLessonsTitleText = 'New Requested Lessons'
+
+export const logOutTitleText = 'Log out'
+
+export const proposedStateDescriptionText = `This lesson is proposed. Waiting for approval.`
+export const cancelledStateDescriptionText = `This lesson has been cancelled.`
+export const acceptedStateDescriptionText = `This lesson has been accepted - it's just waiting to be claimed.`
+export const requestedStateDescriptionText = `This lesson has been requested - it's just waiting to be claimed.`
+export const claimedStateDescriptionText = `This lesson is claimed - it's just waiting for a a video to be submitted.`
+export const submittedStateDescriptionText = 'Sweet! This lesson has been submitted and is waiting for approval.'
+export const rejectedStateDescriptionText = 'This lesson needs updated based on feedback received.'
+export const updatedStateDescriptionText = 'Lesson updated! Waiting for approval.'
+export const approvedStateDescriptionText = 'This lesson has been approved and is in the publishing queue.'
+export const publishedStateDescriptionText = `This lesson has been published. What's next?`
+export const flaggedStateDescriptionText = `This lesson has been flagged - it needs some changes.`
+export const revisedStateDescriptionText = `This lesson has revised. Waiting for approval.`
+export const retiredStateDescriptionText = `This lesson has been retired.`
+
+export const proposeActionText = `Propose`
+export const cancelActionText = `Cancel`
+export const acceptActionText = `Accept`
+export const requestActionText = `Request`
+export const claimActionText = `Claim`
+export const submitActionText = `Submit`
+export const rejectActionText = `Reject`
+export const applyUpdateActionText = `Apply update`
+export const approveActionText =  `Approve`
+export const publishActionText = `Publish`
+export const flagActionText = `Flag`
+export const reviseActionText = `Revise`
+export const retireActionText = `Retire`
+
+export const crackedEggoAltText = 'Cracked eggo logo'
+export const instructorEggoAltText = 'egghead.io instructors logo, eggo only'
+export const instructorBannerAltText = 'egghead.io instructors logo'
```

Comment:
This whole file will be replaced with a real localization library for better in-context hard-coded text and editing